share/spice/justdeleteme/justdeleteme.js

```diff
@@ -0,0 +1,66 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_justdeleteme = function(api_result) {
+        if (!api_result || api_result.length < 1) {
+            return Spice.failed('justdeleteme');
+        }
+
+        // get the remainder
+        var script = $('[src*="/js/spice/justdeleteme/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/justdeleteme\/([^\/]+)/)[1],
+            decodedQuery = decodeURIComponent(query).split(" ")[0].toLowerCase();
+        //         console.log("decodedQuery: " + decodedQuery);
+
+        var isRelevantName = function(item) {
+            if (!item.name)
+                return false;
+            if (item.name.toLowerCase().contains(decodedQuery))
+                return true;   
+            return false;
+        };
+        var isRelevantDomain = function(item) {
```

Comment:
I originally used isRelevant and getRelevants for this, but they got too many false negatives and semeed actually slower.